astropy/io/ascii/tests/test_write.py

```diff
@@ -693,9 +693,10 @@ def test_write_overwrite_ascii(format, fast_writer, tmpdir):
         pass
     t = table.Table([['Hello', ''], ['', '']], dtype=['S10', 'S10'])
 
-    with pytest.raises(OSError) as err:
+    msg = (r'File .* already exists. If you mean to replace it then use the '
+           'argument "overwrite=True".')
```

Comment:
Is this the correct way to escape it? Please double check.

```suggestion
    msg = (r'File .* already exists\. If you mean to replace it then use the '
           'argument "overwrite=True"\.')
```

Reply:
Does it need escaping in a raw string?

Reply:
Yes, otherwise you will get
```python
SyntaxError: invalid escape sequence \.
```